andela/wger_yosemite
# Ft cache nutrition values #147322155

## What does this PR do?

Increase nutrition overview render speed by caching nutrition plan values.

### Description of Task to be completed?

> Configure nutrition app to work with Django signal dispatchers
> Ensure nutrition plan senders(Models) notify receivers any time a change occurs in the models.
> Delete cache every time a change is identified. Changes might be data: addition, deletion or edits.

### How should this be manually tested?

Follow installation guide and run the app
Go into the nutrition tab and click on nutrition plan.
Add a nutrition plan, a meal and meal item.
Every time each of this happens a message is displayed on the console to say cache has been deleted.
True delete meals and nutrition plans, a notification for deletion also shows.
When more than 25 plans exist, total calculation in the overview is faster and so is deletion.

### What are the relevant pivotal tracker stories?
#147322155
